### DISCUSSION TITLE
How to create button with loading icon like login button on login page

### DISCUSSION THREAD

**Discussion by urmatt** [orchidsoftware/platform#1821] [Q&A]
How to create button with loading icon like login button on login page?

I create form that takes for processing the request 8-9 seconds. But at the time it staying like noting happens.

**Comment by moskoweb** [comment]
Here I added the following JavaScript code:
```
jQuery('form').submit(function() {
    var el = jQuery('[type=submit]', jQuery(this));

    el.prop('disabled', true).data('text', el.text()).html('Loading...');
});
```
